### ISSUE TITLE
Add "closed functionality" to Key Terms

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#203]
The term "closed functionality" may not be commonly understood. During the WCAG2ICT TF meeting on 27 July the group came to a unanimous conclusion that this should be added as a key term with a readily understandable definition and some examples.

## Sources to inform the definition
The text definition proposed from the survey was: 
>“Closed functionality" prevents users from attaching, installing, or using their own assistive technology. To support users with disabilities, products with closed functionality can instead provide built-in features that act as assistive technology.

Existing definition from the Revised 508 standards:
>Characteristics that limit functionality or prevent a user from attaching or installing assistive technology. 

Definition from EN 301 549:
>functionality that is limited by characteristics that prevent a user from attaching, installing or using assistive technology 

## Sources of examples of closed functionality products
There are examples from 508: 
>Examples of ICT with closed functionality are self-service machines, information kiosks, set-top boxes, fax machines, calculators, and computers that are locked down so that users may not adjust settings due to a policy such as Desktop Core Configuration.

There are examples from the Canadian standard CSA B651.2 (22):
>This Standard specifies minimum accessibility requirements for self-service interactive devices (such as, but not limited to, automated banking machines, retail self-checkout, self check­ in devices, ticket vending kiosks, smart card sales, query and reload devices).

**Comment by maryjom** [comment]
## Closed Functionality
The term **closed functionality**, as used in WCAG2ICT, has the meaning below:

closed functionality (as used in WCAG2ICT)
functionality that prevents users from attaching, installing, or using [assistive technology](https://wcag2ict.netlify.app/#dfn-assistive-technology) 

NOTE: To support users with disabilities, products with closed functionality can instead provide built-in features that function as assistive technology.

EXAMPLE: Examples of technology with closed functionality include: retail self-checkout machines, automated banking machines (a.k.a. Automated Teller Machines, or ATMs), ticket kiosks, calculators, printers, smart TVs and computers that are locked down so that users may not adjust settings due to a policy.

**Comment by maryjom** [comment]
Question: Is there any need to bring up the topic or describe "partially closed functionality" technology?  I'm thinking that some technologies have some available fully functioning AT built-in, but the built-in AT does not cover all types of AT.

**Comment by loicmn** [comment]
As the survey did not allow me to write comments, let me explain here my view on "partially closed functionality".

A system that has closed functionality does not allow users to attach (i.e. connect physically or wirelessly), install or use assistive technology. And according to the first note such systems should offer accessibility built-in features. 

If these built-in features are incomplete the system still has closed functionality and it is in fact not an accessible system. For instance, the Smart TV I've got at home has a "blind access mode" that reads aloud the menus to provide some accessibility for blind users, but it does not allow to change the fonts to make them bigger, leaving out persons with low vision. And it also does not have an option to remove animations, leaving out people with attention deficit.

To summarize my view, the fact that the built-in accessibility features are incomplete, make this TV not "fully accessible", but does not change the fact that it is a system with closed functionality.

**Comment by maryjom** [comment]
Thanks for noticing that Loïc. Missed the checkbox to add an entry field to the questions. Those are now in place in the survey.

Best regards,

Mary Jo Mueller
IBM Accessibility Standards Program Manager


From: Loïc Martínez Normand ***@***.***>
Date: Wednesday, August 2, 2023 at 12:38 PM
To: w3c/wcag2ict ***@***.***>
Cc: Mary Jo Mueller ***@***.***>, Assign ***@***.***>
Subject: [EXTERNAL] Re: [w3c/wcag2ict] Add "closed functionality" to Key Terms (Issue #203)
As the survey did not allow me to write comments, let me explain here my view on "partially closed functionality". A system that has closed functionality does not allow users to attach (i. e. connect physically or wirelessly), install or use
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
    Report Suspicious  <https://us-phishalarm-ewt.proofpoint.com/EWT/v1/PjiDSg!2g-pItQcolPwkMXw8unGZfXnzPxzAzJ5YTqpc4K6MaIuZoJC6UdISmrUOMElAUQlasHBPBgc0xfpyEoqzafmCNFh07LHKxA$>   ‌
ZjQcmQRYFpfptBannerEnd

As the survey did not allow me to write comments, let me explain here my view on "partially closed functionality".

A system that has closed functionality does not allow users to attach (i.e. connect physically or wirelessly), install or use assistive technology. And according to the first note such systems should offer accessibility built-in features.

If these built-in features are incomplete the system still has closed functionality and it is in fact not an accessible system. For instance, the Smart TV I've got at home has a "blind access mode" that reads aloud the menus to provide some accessibility for blind users, but it does not allow to change the fonts to make them bigger, leaving out persons with low vision. And it also does not have an option to remove animations, leaving out people with attention deficit.

To summarize my view, the fact that the built-in accessibility features are incomplete, make this TV not "fully accessible", but does not change the fact that it is a system with closed functionality.

**Comment by maryjom** [comment]
This was added in PR #222. Next step will be AG WG review/approval.

**Comment by maryjom** [comment]
The AG WG is reviewing this as part of the overall Closed functionality review issue #370 which concludes on 28 May.  Any further requested changes will be tracked in that issue or a new issue.
